Jenkinsfile

```diff
@@ -95,6 +95,27 @@ def rpm_test_daos_test = '''me=\\\$(whoami)
                             trap 'set -x; kill -INT \\\$AGENT_PID \\\$COPROC_PID' EXIT
                             OFI_INTERFACE=eth0 daos_test -m'''
 
+def rpm_scan_daos_test = '''pushd /tmp
+                              wget \
+                              http://rfxn.com/downloads/maldetect-current.tar.gz
```

Comment:
What if this `wget` fails?  Wouldn't it be good enough to use the one that was successfully downloaded probably not at all very long ago?  But on that note, why don't we cache this (at least per builder if not globally for all builders, but admittedly that is probably more work than this PR needs to accomplish) to avoid the re-download every time?  In most runs we should just be checking for a newer version and only download it if it is newer.  See. `curl -z`.

Reply:
Testing a change to cache the lmd source tarball until the upstream changes it.